fix TestClientDisconnect_FailedToWriteResponse()

```
03/03 12:31:00.326: *****: [router.(*httpService).registerHandlersV1.RequestHandler.func5:203]: sending request to VVM on somefunc_HandlerPanic is failed: test panic HandlerPanic . Body:
   
  03/03 12:31:00.333: ===: [router.(*httpService).log:101]: Admin HTTP server: starting on 127.0.0.1:38393
  03/03 12:31:00.333: ===: [router.(*httpService).log:101]: HTTP server: starting on 127.0.0.1:45317
  2025/03/03 12:31:00 {"sections":[{"type":"","elements":[
  2025/03/03 12:31:00 {"sections":[{"type":"","elements":[{"IntField":42,"StrField":"str"}
  2025/03/03 12:31:00 client disconnected during sections sending
  03/03 12:31:00.346: ===: [router.(*httpService).log:101]: HTTP server: starting on 127.0.0.1:35789
  03/03 12:31:00.346: ===: [router.(*httpService).log:101]: Admin HTTP server: starting on 127.0.0.1:36089
  03/03 12:31:00.359: ===: [router.(*httpService).log:101]: Admin HTTP server: starting on 127.0.0.1:43945
  03/03 12:31:00.359: ===: [router.(*httpService).log:101]: HTTP server: starting on 127.0.0.1:46091
  03/03 12:31:00.373: ===: [router.(*httpService).log:101]: Admin HTTP server: starting on 127.0.0.1:33393
  03/03 12:31:00.373: ===: [router.(*httpService).log:101]: HTTP server: starting on 127.0.0.1:44763
  2025/03/03 12:31:00 {"sections":[{"type":"","elements":[
  2025/03/03 12:31:00 {"sections":[{"type":"","elements":[{"IntField":42,"StrField":"str"}
  2025/03/03 12:31:00 failed to write response: write tcp 127.0.0.1:44763->127.0.0.1:35774: write: broken pipe 
   goroutine 91 [running]:
  runtime/debug.Stack()
  	/opt/hostedtoolcache/go/1.23.6/x64/src/runtime/debug/stack.go:26 +0x67
  github.com/voedger/voedger/pkg/router.writeResponse({0xbc3ac8, 0xc0001900e0}, {0xbbafb0, 0x1})
  	/home/runner/work/voedger/voedger/pkg/router/utils.go:31 +0xfb
  github.com/voedger/voedger/pkg/router.reply({0xbc4278, 0xc000134230}, {0xbc3ac8, 0xc0001900e0}, 0xc0001122a0, 0xc0002f4080, {0xaf659e, 0x10}, 0xc0002f4060, 0x0)
  	/home/runner/work/voedger/voedger/pkg/router/impl_reply.go:95 +0x36c
  github.com/voedger/voedger/pkg/router.(*httpService).registerHandlersV1.RequestHandler.func5({0xbc3ac8, 0xc0001900e0}, 0xc0003eb2c0)
  	/home/runner/work/voedger/voedger/pkg/router/impl_http.go:214 +0x536
  net/http.HandlerFunc.ServeHTTP(0xc0000486c0, {0xbc3ac8, 0xc0001900e0}, 0xc0003eb2c0)
  	/opt/hostedtoolcache/go/1.23.6/x64/src/net/http/server.go:2[220](https://github.com/voedger/voedger/actions/runs/13630905066/job/38098280019?pr=3405#step:8:222) +0x48
  github.com/voedger/voedger/pkg/router.(*httpService).registerHandlersV1.corsHandler.func6({0xbc3ac8, 0xc0001900e0}, 0xc0003eb2c0)
  	/home/runner/work/voedger/voedger/pkg/router/impl_http.go:228 +0x38f
  net/http.HandlerFunc.ServeHTTP(0xc000208918, {0xbc3ac8, 0xc0001900e0}, 0xc0003eb2c0)
  	/opt/hostedtoolcache/go/1.23.6/x64/src/net/http/server.go:[222](https://github.com/voedger/voedger/actions/runs/13630905066/job/38098280019?pr=3405#step:8:224)0 +0x48
  github.com/gorilla/mux.(*Router).ServeHTTP(0xc0004da240, {0xbc3ac8, 0xc0001900e0}, 0xc0003eb040)
  	/home/runner/go/pkg/mod/github.com/gorilla/mux@v1.8.1/mux.go:212 +0x372
  net/http.serverHandler.ServeHTTP({0xc0001d41b0?}, {0xbc3ac8, 0xc0001900e0}, 0xc0003eb040)
  	/opt/hostedtoolcache/go/1.23.6/x64/src/net/http/server.go:3210 +0x2a2
  net/http.(*conn).serve(0xc000194090, {0xbc4240, 0xc0004167b0})
  	/opt/hostedtoolcache/go/1.23.6/x64/src/net/http/server.go:2092 +0x12a5
  created by net/http.(*Server).Serve in goroutine 139
  	/opt/hostedtoolcache/go/1.23.6/x64/src/net/http/server.go:3360 +0x8ed
  
  2025/03/03 12:31:00 client disconnected during sections sending
  --- FAIL: TestClientDisconnect_FailedToWriteResponse (0.03s)
      impl_test.go:338: 
          	Error Trace:	/home/runner/work/voedger/voedger/pkg/router/impl_test.go:338
          	Error:      	Received unexpected error:
          	            	context canceled
          	Test:       	TestClientDisconnect_FailedToWriteResponse
  03/03 12:31:00.404: ===: [router.(*httpService).log:101]: Admin HTTP server: starting on 127.0.0.1:45651
  03/03 12:31:00.404: ===: [router.(*httpService).log:101]: HTTP server: starting on 127.0.0.1:35013
  2025/03/03 12:31:00 dial tcp 10.1.0.114:45651: connect: connection refused
  FAIL
  FAIL	github.com/voedger/voedger/pkg/router	0.155s


```

--- a/pkg/router/impl_test.go
+++ b/pkg/router/impl_test.go
@@ -313,7 +313,7 @@ func TestClientDisconnect_FailedToWriteResponse(t *testing.T) {
 				StrField: "str1",
 			})
 		}()
-	}, bus.SendTimeout(time.Hour)) // one hour timeout to eliminate case when client context closes longer than bus timoeut on client disconnect. It could take up to few seconds
+	}, bus.SendTimeout(time.Hour)) // one hour timeout to eliminate case when client context closes longer than bus timeout on client disconnect. It could take up to few seconds
 	defer tearDown(router)
 
 	// client side

--- a/pkg/router/impl_test.go
+++ b/pkg/router/impl_test.go
@@ -294,24 +294,23 @@ func TestClientDisconnect_FailedToWriteResponse(t *testing.T) {
 				StrField: "str",
 			})
 
-			// now let's wait for client disconnect
+			// now let's wait for setting onBeforeWriteResponse hook
 			<-setDisconnectOnWriteResponse
 
+			// next send to bus should be successful because client will be disconnected on the next writeResponse()
+			// this operation should trigger the client disconnect
 			expectedErrCh <- respWriter.Write(testObject{
 				IntField: 42,
 				StrField: "str0",
 			})
 
-			// this object must be successfully sent to bus but the router will fail to send in on next writeResponse() call
+			// wait for the client disconnection
+			<-requestCtx.Done()
+
+			// next sending to bus must be failed because the ctx is closed
 			expectedErrCh <- respWriter.Write(testObject{
 				IntField: 43,
 				StrField: "str1",
-			})
-
-			// this sending to bus must be failed because requestCtx stored in IResponseSender is closed
-			expectedErrCh <- respWriter.Write(testObject{
-				IntField: 44,
-				StrField: "str2",
 			})
 		}()
 	}, bus.SendTimeout(time.Hour)) // one hour timeout to eliminate case when client context closes longer than bus timoeut on client disconnect. It could take up to few seconds
@@ -337,14 +336,7 @@ func TestClientDisconnect_FailedToWriteResponse(t *testing.T) {
 	// force client disconnect right before write to the socket on the next writeResponse() call
 	once := sync.Once{}
 	onBeforeWriteResponse = func(w http.ResponseWriter) {
-		once.Do(func() {
-			resp.Body.Close()
-
-			// wait for write to the socket will be failed indeed. It happens not at once
-			// that will guarantee context.Canceled error on next sending instead of possible ErrNoConsumer
-			for _, err := w.Write([]byte{0}); err == nil; _, err = w.Write([]byte{0}) {
-			}
-		})
+		once.Do(func() { resp.Body.Close() })
 	}
 
 	// signal to the handler it could try to send the next section
@@ -352,8 +344,7 @@ func TestClientDisconnect_FailedToWriteResponse(t *testing.T) {
 	// but will be failed in router (will be disconnected right on writeResponse)
 	close(setDisconnectOnWriteResponse)
 
-	// first elem send after client disconnect should be successful, next one should fail
-	require.NoError(<-expectedErrCh)
+	// first elem send after client disconnect should be successful
 	require.NoError(<-expectedErrCh)
 
 	// next sending to the bus must be failed because the requestCtx is closed

